src/main/java/org/jvnet/hudson/plugins/thinbackup/ThinBackupPluginImpl.java

```diff
@@ -60,40 +68,61 @@ public class ThinBackupPluginImpl extends Plugin {
     private boolean backupBuildsToKeepOnly = false;
     private boolean failFast = true;
 
-    @Override
-    public void start() throws Exception {
-        super.start();
+    @DataBoundConstructor
+    public ThinBackupPluginImpl() {
+        // check if old config is there and no new config exists
+        final File oldConfig = new File(Jenkins.get().getRootDir(), "thinBackup.xml");
+        final File newConfig = getConfigFile().getFile();
+        boolean oldConfigExists = oldConfig.exists();
+        boolean newConfigExits = newConfig.exists();
+        if (oldConfigExists && !newConfigExits) {
+            LOGGER.warning("old config of 'thinBackup' detected, moving to new name");
+            try {
+                Files.move(oldConfig.toPath(), newConfig.toPath(), StandardCopyOption.REPLACE_EXISTING);
+            } catch (IOException e) {
+                LOGGER.severe(
```

Comment:
Perhaps `throw new UncheckedIOException(e);` would be more desirable than logging and driving on, following the general principle of failing fast, but no strong preference here.